src/main/tv/codely/scala_http_api/module/course/domain/CourseLessons.scala

```diff
@@ -0,0 +1,3 @@
+package tv.codely.scala_http_api.module.course.domain
+
+case class CourseLessons(value: BigDecimal)
```

Comment:
Maybe an `Int` would be enough. We don't expect having courses with more than 15 lessons :P